docs/GraFx-Studio/concepts/design-run/index.md

```diff
@@ -0,0 +1,63 @@
+# Design Mode and Run Mode
+
+GraFx Studio provides two modes for creating and testing Smart Templates: **Design Mode** and **Run Mode**. 
+
+These modes allow you to focus separately on designing the smart template and testing business logic.
+
+In **Design Mode**, you can freely arrange and style elements without triggering actions or interactions. This keeps the workspace clean, ideal for setting up and adjusting layouts, styles, and initial logic.
+
+Switching to **Run Mode** simulates the end-user experience (My Projects), showing how the template’s logic behaves live. In this mode, actions and visibility settings behave as they will in a real environment, letting you test how users will experience the template.
```

Comment:
We don't have visibility settings yet.

Reply:
I'm referring to visibility of variables, and also visibility of layers, depending on the chosen layout. I'll make it more clear